internal/api/auth.go

```diff
@@ -281,10 +338,16 @@ func (client *Auth) GetTokenForScopes(ctx context.Context, scopes []string) (*st
 		token, tokenExpiry, err = client.AuthenticateClientSecret(ctx, scopes)
 	case client.config.IsCliProvided():
 		token, tokenExpiry, err = client.AuthenticateUsingCli(ctx, scopes)
+	case client.config.IsAzDOWorkloadIdentityFederationProvided():
```

Comment:
Do I get that right? Now we have a dedicated AzDO setup but for GitHub one has to use generic Oidc auth? Maybe we should mention that somewhere because now it looks that github is not supported at all?

Reply:
cool